activestorage/active.py

```diff
@@ -111,16 +191,48 @@ def __from_storage(self, stripped_indexer, drop_axes, out_shape, out_dtype, fsre
             self._process_chunk(fsref, chunk_coords,chunk_selection, out, out_selection,
                                     drop_axes=drop_axes)
 
-        if self.method is not None:
-            return self.method(out)
-        else:
-            return out
+        if method is not None:
+            out = method(out)
+            if self._components:
+                # Return a dictionary of components containing the
+                # reduced data and the sample size ('n'). (Rationale:
+                # cf-python needs the sample size for all reductions;
+                # see the 'mtol' parameter of cf.Field.collapse.)
```

Comment:
are we assuming usage of cf-python?

Reply:
No, but that it is a use case that would be good to have, since we'll be showcasing this in cf-python, I think. (We need 'n' for the mean, in any event, so it's not too much of a leap to include it for each method.)

Reply:
sounds good to me!

Reply:
cheers :beer: